ExperimentRunner/Plugins/Batterystats.py

```diff
@@ -82,8 +82,8 @@ def collect_results(self, device, path=None):
         # charge is given in mAh
         charge = device.shell('dumpsys batterystats | grep "Computed drain:"').split(',')[1].split(':')[1]
         volt = device.shell('dumpsys batterystats | grep "volt="').split('volt=')[1].split()[0]
-        energy_consumed_Wh = float(charge) * float(volt) / 1000.0
-        energy_consumed_J = energy_consumed_Wh * 3600.0
+        energy_consumed_Wh = float(charge) * float(volt) / 1000.0 #volt are in millivolt, charge in mAh, here ther's a division by 1000 => Wh is wrong, it is in mWh
+        energy_consumed_J = energy_consumed_Wh * 3600.0 #J is wrong, is in mJ
```

Comment:
instead of commenting the line of code, change the name of the variable and rename the column of the produced csv file.

Reply:
I fixed it, now it' is ok. 
PS another change is in converting Wh to Joule, 1 Wh = 3,6*10^3 J => 1Wh/3,6*10^3 =1 J

Reply:
Ok, now it is ok. i put a division by 1000000 at line 85 (1000 for mAh and 1000 for mV)